source/clear-linux/get-started/compatibility-check.rst

```diff
@@ -3,26 +3,25 @@
 Check processor and EFI firmware compatibility with Clear Linux OS for Intel® Architecture
 ======================================================================
 
-On a system that is currently running a Linux operating system, follow the instructions below to determine if your target system is capable of running |CL|\*.  Otherwise, see the Note section below for other options.  
+On a system that is currently running a Linux operating system, follow the instructions below to determine if your target system is capable of running |CL|.  Otherwise, run |CL| as a `live image`_ and perform the steps below.  
 
-#. Download the `clear-linux-check-config.sh`_ script.
+#. Download the `clear-linux-check-config.sh file`_ script.
```

Comment:
This was a link, referred to at the bottom of the file. Was the intent here to replace "script" with "file"?